sphinx-markdown-2021.rst

```diff
@@ -0,0 +1,115 @@
+.. post:: Dec 16, 2021
+   :tags: media, talk, conference, markdown, commonmark
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      During 2021 we have promoted the use of Markdown in Sphinx in various ways.
+      In this post we show how you can start using it in your project today,
+      and give some extra pointers.
+
+Sphinx and Markdown around the world in 2021
+============================================
+
+Read the Docs has been committed to improving the accessibility
+and user experience of Sphinx since its beginnings,
+and that includes the markup language in which the documentation is written.
+After carefully listening to users
+that wanted to use Markdown in Sphinx as well as reStructuredText,
+:doc:`we announced recommonmark more than six years ago </adding-markdown-support>`
+to help bridging the immense popularity of Markdown
+with the powerful capabilities of Sphinx.
```

Comment:
I found this sentence confusing when I read it. At first, I thought we listen to our users in the last months, but then I realized that it's taking about 6 years ago. However, I've already clicked on the blog post and I was reading it 😄 

If possible, I'd change the order of it to mention the time at the beginning of the sentence to immediately contextualize the reader. Something like,

```
Years ago, after carefully listening to users ... we announced recommonmark
```

Besides, I think it would be good to mention in the same paragraph that it's currently deprecated, something like:

```
which is now, deprecated in favor of MyST (keep reading to find out more about how to use it / migrate)
```

Reply:
Yea, 👍 to mentioning that it's deprecated in the same sentence.

Reply:
Done ✔️